src/data/real/ennreal.lean

```diff
@@ -92,13 +92,6 @@ localized "notation (name := ennreal.top) `∞` := (⊤ : ennreal)" in ennreal
 namespace ennreal
 variables {a b c d : ℝ≥0∞} {r p q : ℝ≥0}
 
--- TODO: why are the two covariant instances necessary? why aren't they inferred?
-instance covariant_class_mul_le : covariant_class ℝ≥0∞ ℝ≥0∞ (*) (≤) :=
-canonically_ordered_comm_semiring.to_covariant_mul_le
-
-instance covariant_class_add_le : covariant_class ℝ≥0∞ ℝ≥0∞ (+) (≤) :=
-ordered_add_comm_monoid.to_covariant_class_left ℝ≥0∞
-
```

Comment:
These ones _might_ be worth keeping as shortcuts, does the git blame reveal why they were added?

Reply:
Looks like they were moved in #9891, which is probably long enough ago not to care